qiskit_dynamics/backend/dynamics_backend.py

```diff
@@ -73,8 +74,15 @@ class DynamicsBackend(BackendV2):
       outcome probabilities. Defaults to ``True``. Setting to ``False`` can result in errors if
       the solution tolerance results in probabilities with significant numerical deviation from
       proper probability distributions.
-    * ``meas_level``: Form of measurement return. Only supported value is ``2``, indicating that
-      counts should be returned. Defaults to ``meas_level==2``.
+    * ``meas_level``: Form of measurement output. Suported values are ``1`` and ``2``.
+      ``1`` returns IQ points and ``2`` returns counts. Defaults to ``meas_level==2``.
+    * ``meas_return``: Level of measurement data to return. ``"single"`` returns information from
```

Comment:
This only impacts `meas_level == 1` right? If yes can you make this doc line state this?

Reply:
yeah